alfalfa_worker/lib/alfalfa_connections_base.py

```diff
@@ -47,7 +46,7 @@ def __init__(self):
             self.influx_db_name = None
             self.influx_client = None
 
-    def add_site_to_mongo(self, haystack_json, site_ref):
+    def add_site_to_db(self, haystack_json, site_ref):
```

Comment:
Is this being called from anywhere?

Reply:
It's called in two places:
- [Modelica](https://github.com/NREL/alfalfa/blob/2057c8829a869a89bda6c245ecbddb3e69576f4a/alfalfa_worker/jobs/modelica/create_run.py#L70)
- [OpenStudio](https://github.com/NREL/alfalfa/blob/2057c8829a869a89bda6c245ecbddb3e69576f4a/alfalfa_worker/jobs/openstudio/create_run.py#L131)

Reply:
But that calls [this function](https://github.com/NREL/alfalfa/blob/2057c8829a869a89bda6c245ecbddb3e69576f4a/alfalfa_worker/lib/run_manager.py#L182) not the one in alfalfa_connections_base